### PR TITLE
Sao 90 config bundle

### PR DESCRIPTION
**Pull request by kmadel**
Ready for review.

### CODE REVIEW

**Comment by logandonley** [comment]
Reviewing this morning

**Comment by logandonley** [comment]
We'll need to be sure to prebake these script approvals:
```
method hudson.util.Secret getEncryptedValue
method java.util.Dictionary get java.lang.Object
new hudson.util.Secret java.lang.String
staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods getProperties java.lang.Object
```

**Comment by logandonley** [comment]
@kmadel Could we parameterize the core namespace? The helm installer uses `cloudbees-core` by default.

File: templates/cb-days-setup/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            git push -u origin master
+        """)
+        container('kubectl') {
+          sh "mkdir ${masterName}"
+          sh "cp *.yaml ${masterName}"
+          sh "kubectl cp --namespace core ${masterName} cjoc-0:/var/jenkins_home/jcasc-bundles-store/"
+          sh "kubectl exec --namespace core cjoc-0 -- sed -i \"/<\\/access>/i\\<entry><string>${masterName}</string><hudson.util.Secret>${entrySecret}</hudson.util.Secret></entry>\"  /var/jenkins_home/jcasc-bundles-store/security.xml"
+        }
+        echo "preparing Jenkins CLI for Master restart"
+        sh 'curl -O http://managed-masters-ops.cje.svc.cluster.local/managed-masters-ops/jnlpJars/jenkins-cli.jar'
```
Comment (logandonley) [inline templates/cb-days-setup/Jenkinsfile]: Does this ops mm need to have anything specific on it? Also, it's hardcoded to the cje namespace.

Comment (kmadel) [reply templates/cb-days-setup/Jenkinsfile]: Regarding script approvals, the image here has been updated: https://github.com/kypseli/cb-core-mm-workshop/blob/preview/jenkins_ref/scriptApproval.xml

Agree about the K8s namespace, needs to be parameterized.

Comment (kmadel) [reply templates/cb-days-setup/Jenkinsfile]: Also the op mm needs a username password credential with an id of `cli-username-token` with the username being the Core Jenkins username of an account that has admin privileges on OC and the password being that users Jenkins API token. I have been storing this credential in the folder on OC that contains the ops master.

**Pull request closed without merging**